.gitignore

```diff
@@ -2,3 +2,43 @@
 /shelf/
 /workspace.xml
 /.idea
+
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
+
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
+
+/public/assets
+.byebug_history
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
+
+
+/config/database.yml
```

Comment:
Add a new line at the end of the file. Github displays an icon at the last line to highlight the issue.
Read more info in https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
It is possible that you can configure your editor to add the empty line automatically. Check the editor setting for more info.